### ISSUE TITLE
NPM package is broken when installing from a github url

### ISSUE THREAD

**Issue by varb** [akiran/react-slick#1015]
I made fork from this repo, cloned to local machine and changed slider for my project. Then pushed to my repo and I wanted to add to my project.
At first I tried install `react-slick` from origin repo by
```bash
npm i -S react-slick
```
In `node_modules` folder appeared two important folder `dist` and `lib` and I was able to connect `react-slick` in my React app.

Ok. Then I tried install my fork:
```bash
npm i -S varb/react-slick
```
And in `node modules` folder there were no two folders `dist` and `lib`

I found a similar question on [stack overflow](https://stackoverflow.com/questions/35856238/npm-installing-forked-respository-with-custom-changes-does-not-build-the-modul), but there is no clear answer to it.

So far, I'm using build of slider from the `dist` folder, putting the file directly into the project, bypassing npm

**Comment by akiran** [comment]
This command generates lib and dist folder
https://github.com/varb/react-slick/blob/master/package.json#L12

If you want to install from github, you can change prepublish to postinstall in that line

**Comment by varb** [comment]
@akiran thanks for answer!

I changed `prepublish` on `postinstall` and tried again `npm i -S varb/react-slick`.
At first I got a message `sh: 1: babel: not found` and install was failed. 
I installed babel by `npm i -D babel-cli babel-preset-env` and launched install again.
Now installation fails with message
```bash
> babel ./src --out-dir ./lib && gulp dist

./src doesn't exist
```

How to make that when I trying install plugin from my repo to load folder `src`?
P.S.: Sorry for mistakes in my English
